proxy/src/main/java/com/wavefront/agent/PointHandlerImpl.java

```diff
@@ -33,15 +33,11 @@
   private static final Logger blockedPointsLogger = Logger.getLogger("RawBlockedPoints");
   private static final Logger validPointsLogger = Logger.getLogger("RawValidPoints");
 
-  /**
-   * Executor for support tasks (refreshing logger level)
-   */
-  private final ScheduledExecutorService pointHandlerExecutor;
-
   private final Histogram receivedPointLag;
   private final String validationLevel;
   private final String handle;
   private boolean logPoints = false;
+  private long logPointsUpdatedMillis = 0L;
```

Comment:
volatile to be absolutely correct. setting the value of a long might not be atomic

Reply:
oh you did it later